src/SignalR/server/Core/src/Internal/DefaultHubDispatcher.cs

```diff
@@ -284,7 +284,7 @@ private Task ProcessStreamItem(HubConnectionContext connection, StreamItemMessag
             var methodExecutor = descriptor.MethodExecutor;
 
             var disposeScope = true;
-            var scope = _serviceScopeFactory.CreateScope();
+            await using var scope = _serviceScopeFactory.CreateAsyncScope();
```

Comment:
remove `await using`, this scope has a complex lifetime and is disposal is handled elsewhere